app/services/resource_service.rb

```diff
@@ -15,101 +15,159 @@ def create(resource_class, params)
     # Erlaube nur ein Anlegen von Daten wenn der Nutzer nicht ReadOnly ist.
     raise "Access not permitted" if data_provider.user.read_only_role?
 
+    # cleanup params for given :id
```

Comment:
`def create resource_class, params)` => `def perform(resource_class, params)` und alle aufrufe anpassen

Reply:
done in a02fcfa